tests_metricflow/snapshots/test_custom_granularity.py/SqlQueryPlan/DuckDB/test_offset_metric_with_custom_granularity__plan0_optimized.sql

```diff
@@ -2,11 +2,8 @@ test_name: test_offset_metric_with_custom_granularity
 test_filename: test_custom_granularity.py
 sql_engine: DuckDB
 ---
--- Compute Metrics via Expressions
-SELECT
-  booking__ds__martian_day
-  , bookings_5_days_ago AS bookings_5_day_lag
-FROM (
+-- Read From CTE For node_id=cm_5
+WITH cm_4_cte AS (
```

Comment:
What does cm mean?

Reply:
That's the internal ID prefix used for the dataflow node that computes metrics.